CONTRIBUTING.MD

```diff
@@ -0,0 +1,64 @@
+Lyra Contributing Guidelines
+=======================
+
+New to Lyra? Check out "First PR" tag in the issues. These are issues have a lower level of effort for people looking to get their hands dirty in the Lyra codebase.
+
+## Guideline Contents
+
+There are a lot of ways to get involved and help out:
+
+- [Reporting an issue](#reporting-issues)
+- [Requesting features](#requesting-features)
+- [Submitting Pullrequests](#submitting-prs)
+- [Writing Documentation & tutorials](#documentation)
+- [Contributing Design Assets](#contributing-design)
```

Comment:
I guess the anchors do de-couple the wording of the titles from this ToC so disregard my earlier comment :)